README.md

```diff
@@ -113,10 +136,7 @@ npm run test:watch
 
 We use [Open API 3.0](https://www.openapis.org/about) to define the API structure of the application. You can see the full documentation with:
```

Comment:
nit: combine this line and the line below since we're no longer asking them to run a command